ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -71,16 +71,15 @@ export const appRoutes: Routes = [
                 component: DefaultLayoutComponent,
                 children: [
                     {path: '', redirectTo: "/project-list", pathMatch: "full"},
+                    {path: 'about', component: AboutPageComponent},
                     {
                         path: "configuration",
-                        resolve: { configuration: ConfigurationResolve },
```

Comment:
Well, there are multiple, but the question is - if we are so obsessed with consistency, why was the file in that state in the first place? If it weren't, IDEA wouldn't reformat that. I just didn't notice.